meerk40t/core/cutcode/rastercut.py

```diff
@@ -122,8 +123,8 @@ def length(self):
             scan_distance = self.width * scan_stride
         else:
             scanlines = self.width
-            scan_stride = self.step_x
-            scan_step = self.step_y
+            scan_stride = self.step_y
+            scan_step = self.step_x
             scan_distance = self.height * scan_stride
```

Comment:
Personally I think that these lines should be in the opposite sequence so that it makes it easier to compare with lines 121-122 i.e. left side is in the same sequence and right side is reversed for a raster in the opposite direction.